content/en/docs/concepts/storage/volumes.md

```diff
@@ -62,96 +60,23 @@ a different volume.
 
 Kubernetes supports several types of volumes.
 
-### awsElasticBlockStore (removed) {#awselasticblockstore}
+The following volume types are deprecated.
 
```

Comment:
I would add something like:
```html

<a id="azureDisk"></a>
<a id="gcepersistentdisk"></a>

```
so that existing hyperlinks go to approximately the right place.

Reply:
Emm ... these anchors are automatically added when the Markdown is processed.

Reply:
When we delete
```markdown
### awsElasticBlockStore (removed) {#awselasticblockstore}
```
then I don't think the `#awselasticblockstore` fragment identifier will still work. Did I miss something?